nservicebus/upgrades/release-policy.md

```diff
@@ -100,7 +100,6 @@ The release cycle consists of the following quality stages:
 
 ### Beta
 
-* The component is feature complete.
```

Comment:
Instead of removing this entirely, would it be better to state something similar to the RC section? Something like:

In this stage of stabilization, all features have been designed and coded but may lack functionality and could contain significant bugs or performance issues.

Reply:
Without enforcing artificial code freezes once a beta starts, we can't enforce that all features have been designed and coded. The only way to enforce that is to branch a release branch at beta